packages/shorebird_cli/test/src/commands/preview_command_test.dart

```diff
@@ -188,15 +190,37 @@ void main() {
               authRef.overrideWith(() => auth),
               bundletoolRef.overrideWith(() => bundletool),
               cacheRef.overrideWith(() => cache),
-              codePushClientWrapperRef
-                  .overrideWith(() => codePushClientWrapper),
+              codePushClientWrapperRef.overrideWith(
+                () => codePushClientWrapper,
+              ),
               loggerRef.overrideWith(() => logger),
               shorebirdValidatorRef.overrideWith(() => shorebirdValidator),
             },
           ),
         );
       }
 
+      void setupExtractArtifactStub() {
```

Comment:
I'm struggling to come up with a better name, but it's not totally clear to me without reading the body of the function what this does. Maybe the `thenAnswer` body can be pulled into a separate function (`writeShorebirdYamlToAssetsDir` or something) and this function can be inlined below?